app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb

```diff
@@ -1,4 +1,18 @@
 module ManageIQ::Providers::Redhat::InfraManager::Vm::Operations::Snapshot
+  extend ActiveSupport::Concern
+
+  included do
+    supports :snapshots do
+      unless ext_management_system&.supports_snapshots?
```

Comment:
Travis is ok with this, its only rubocop, and once its merged it will not raise a warning again until its changed again.